dolphinscheduler-ui/src/views/security/cluster-manage/index.tsx

```diff
@@ -0,0 +1,170 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { defineComponent, onMounted, toRefs, watch } from 'vue'
+import {
+  NButton,
+  NCard,
+  NDataTable,
+  NIcon,
+  NInput,
+  NPagination
+} from 'naive-ui'
+import { SearchOutlined } from '@vicons/antd'
+import { useI18n } from 'vue-i18n'
+import { useTable } from './use-table'
+import Card from '@/components/card'
+import ClusterModal from './components/cluster-modal'
+import styles from './index.module.scss'
+
+const clusterManage = defineComponent({
+  name: 'cluster-manage',
+  setup() {
+    const { t } = useI18n()
+    const { variables, getTableData, createColumns } = useTable()
+
+    const requestData = () => {
+      getTableData({
+        pageSize: variables.pageSize,
+        pageNo: variables.page,
+        searchVal: variables.searchVal
+      })
+    }
+
+    const onUpdatePageSize = () => {
+      variables.page = 1
+      requestData()
+    }
+
+    const onSearch = () => {
+      variables.page = 1
+      requestData()
+    }
+
+    const handleModalChange = () => {
+      variables.showModalRef = true
+      variables.statusRef = 0
+    }
+
+    const onCancelModal = () => {
+      variables.showModalRef = false
+    }
+
+    const onConfirmModal = () => {
+      variables.showModalRef = false
+      requestData()
+    }
+
+    onMounted(() => {
+      createColumns(variables)
+      requestData()
+    })
+
+    watch(useI18n().locale, () => {
+      createColumns(variables)
+    })
+
+    return {
+      t,
+      ...toRefs(variables),
+      requestData,
+      onCancelModal,
+      onConfirmModal,
+      onUpdatePageSize,
+      handleModalChange,
+      onSearch
+    }
+  },
+  render() {
+    const {
+      t,
+      requestData,
+      onUpdatePageSize,
+      onCancelModal,
+      onConfirmModal,
+      handleModalChange,
+      onSearch,
+      loadingRef
+    } = this
+
+    return (
+      <div>
+        <NCard>
+          <div class={styles['search-card']}>
+            <div>
+              <NButton
+                size='small'
+                type='primary'
+                onClick={handleModalChange}
+                class='btn-create-cluster'
+              >
+                {t('security.cluster.create_cluster')}
+              </NButton>
+            </div>
+            <div class={styles.box}>
+              <NInput
+                size='small'
+                clearable
+                v-model={[this.searchVal, 'value']}
+                placeholder={t('security.cluster.search_tips')}
+              />
+              <NButton size='small' type='primary' onClick={onSearch}>
+                {{
+                  icon: () => (
+                    <NIcon>
+                      <SearchOutlined />
+                    </NIcon>
+                  )
+                }}
+              </NButton>
+            </div>
+          </div>
```

Comment:
It is better to use 'NSpace' component here.

Reply:
Code changed, thank you.

Reply:
Sorry, maybe I didn't express clearly. I mean that using NSpace instead of the div with 'search-card' class. And then you don't need write css.

Reply:
Hi  @Amy0104,  yes, you are right, for this part, I have changed the code with NSpace(without css), it's better use NSpace.
And about the next comments, lines near 155, as the parent div also has margin-top, not only keep NPagination center, so I kept the same code(<div class={styles.pagination}>) with other pages, because they have implemented the unified css margin-top in css file

Reply:
<img width="1101" alt="image" src="https://user-images.githubusercontent.com/13530192/173729266-f3235138-c58a-43d2-9c54-afeea1d680d1.png">
<img width="580" alt="image" src="https://user-images.githubusercontent.com/13530192/173729291-6a97a9ad-f4c6-498c-90e4-f0e3f79334ac.png">

Reply:
@Amy0104

Reply:
I have changed it. Pls, check.

Reply:
Sure, thank you for your update. BTW, the style between pages is best to be uniform, I found that there is a small place that is not the same as other pages.

Reply:
<img width="1755" alt="image" src="https://user-images.githubusercontent.com/13530192/173732007-b98e3edf-ca77-4c8a-93ca-320e8276b617.png">
<img width="1751" alt="image" src="https://user-images.githubusercontent.com/13530192/173732091-7e2694f5-e207-43cb-a9d4-886b65988f23.png">

Reply:
There are a few small differences between pages, probably written by different developers.
<img width="1757" alt="image" src="https://user-images.githubusercontent.com/13530192/173732529-fa261600-37d2-4259-9bc4-eeca120414de.png">
<img width="1756" alt="image" src="https://user-images.githubusercontent.com/13530192/173732566-8c862585-f4e1-47c0-af0d-56aee234a58f.png">
<img width="1778" alt="image" src="https://user-images.githubusercontent.com/13530192/173732703-96c0f0cc-4ade-477e-bf4b-7ca08c4462de.png">
<img width="1734" alt="image" src="https://user-images.githubusercontent.com/13530192/173732750-dfd955a6-4dd9-4790-898c-0635161e9103.png">

Reply:
That's true. So it's better to use Nspace when we need a space in a page.